### PR TITLE
Add Huly Platform

### PR DESCRIPTION
**Pull request by aplatoff**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/huly.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "Huly"
+website_url: "https://huly.io"
+source_code_url: "https://github.com/hcengineering/platform"
+description: "All-in-One Replacement for Linear/Jira, Slack, Notion, Motion and more. Integrated with GitHub and others."
```
Comment (nodiscc) [inline software/huly.yml:4]: Hi,

The description should explain what the software does.

You can use the following syntax to denote the relationship to Linea/Jira (https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines)

> If the project is presented as an alternative to another service or application, please mention it as (alternative to $PRODUCT1, $PRODUCT2) at the end of the description.

**Review by nodiscc — DISMISSED** [review]

File: software/huly.yml (hunk 1 above)
Comment (aplatoff) [reply software/huly.yml]: Thank you, fixed!

File: software/huly.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - Communication - Custom Communication Systems
+  - Document Management
+  - Groupware
+  - Human Resources Management (HRM)
+  - Knowledge Management Tools
+  - Note-taking & Editors
+  - Task Management & To-do Lists
+  - Ticketing
+  - Time Tracking
+  - Wikis
```
Comment (nodiscc) [inline software/huly.yml:25]: Please don't try to abuse the tagging system...

```suggestion
tags:
  - Software Development - Project Management
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]
CI failure is not related to this PR.
Thanks

**Pull request merged**